core/app/c/[communitySlug]/pubs/page.tsx

```diff
@@ -33,6 +34,11 @@ export default async function Page(props: Props) {
 
 	return (
 		<>
+			<PubsDataTableServer
+				communityId={community.id}
+				userId={user.id}
+				searchParams={searchParams}
+			/>
```

Comment:
Plopped this in the all pubs page just as an example. Seems to work decently well, with the caveat that the table is only rendering title atm. would probably be more complicated once we want it to start rendering action menus

![image](https://github.com/user-attachments/assets/5658572b-c636-4a0a-8365-5e1882253cce)